dev/src/path.ts

```diff
@@ -455,7 +455,7 @@ export class QualifiedResourcePath extends ResourcePath {
     if (elements) {
       const project = elements[1];
       const database = elements[2];
-      const path = elements[3];
+      const path = elements[3] ?? '';
```

Comment:
Nit: If we know path is empty, then I think we should avoid calling append (even it it supports it).

Reply:
Yeah, nice improvement. I've updated this.